packages/shared/src/contracts/ethers-typechain/factories/GlobalAllowList__factory.ts

```diff
@@ -0,0 +1,546 @@
+/* Autogenerated file. Do not edit manually. */
+/* tslint:disable */
+/* eslint-disable */
+
+import { Contract, Signer, utils } from "ethers";
+import type { Provider } from "@ethersproject/providers";
+import type {
+  GlobalAllowList,
+  GlobalAllowListInterface,
+} from "../GlobalAllowList";
+
+const _abi = [
```

Comment:
is there a difference between this `_abi` and `packages/shared/abis/GlobalAllowList.json`?

Reply:
This is code generated by `typechain`, the difference includes generated (inferred) types.

Reply:
ok cool, so as one changes, the other will be automatically kept up to date by typechain